whitepaper/whitepaper.md

```diff
@@ -701,6 +701,48 @@ Hyperledger Cactus has several integration patterns as the following.
 | 4.  | data transfer       | D -> D  | check if all D1 is copied on ledger 2 <br> (as D1 is data on ledger 1, D2 is data on ledger 2) |
 | 5.  | data merge          | D <-> D | check if D1 = D2 as a result <br> (as D1 is data on ledger 1, D2 is data on ledger 2)          |
 
+## 5.1.1 Deployment Scenarios
+
+There's a near-infinite possible deployment scenarios which is intended to be one of the benefits of using Cactus where
+you are not constrained by strong opinions imposed upon you by the framework but instead can tailor it to your needs
+while still getting value out of it.
+There's a set of building blocks (members, nodes, API server processes, plugin instances) that you can use when defining (founding) a consortium and these building blocks relate to each other in a way that can be expressed with an entity relationship diagram which can be seen below.
+The composability rules can be deducted from how the diagram elements (entities) are connected (related) to each other, e.g. the API server process can have any number of plugin instances in it and a node can contain any number of API server processes, and so on until the top level construct is reached: the consortium.
+
+> Consortium management does not relate to achieving consensus on data/transactions involving individual ledgers, merely about consensus on the metadata of a consortium.
+
+<img width="400" src="./deployment-entity-relationship-diagram.png">
+
+Now, with these composability rules in mind, let us demonstrate a few different deployment scenarios (both expected and exotic ones) to showcase the framework's flexibility in this regard.
+
+### 5.1.1 Production Deployment Example
+
+Many different configurations are possible here as well.
+One way to have two members form a consortium and both of those members provide highly available, high throughput services is to have a deployment as shown on the below figure.
+What is important to note here is that this consortium has 2 nodes, 1 for each member
+and it is irrelevant how many API servers those nodes have internally because they
+all respond to requests through the network host/web domain that is tied to the
+node.
+One could say that API servers do not have a distinguishable identity relative to
+their peer API servers, only the higher-level nodes do.
+
+<img width="700" src="./deployment-production-example.png">
```

Comment:
@petermetz  Can you remove this diagram until we can agree with the deployment model? Then I can approve others.

Reply:
> @petermetz Can you remove this diagram until we can agree with the deployment model? Then I can approve others.

@sfuji822 https://github.com/hyperledger/cactus/pull/331#issuecomment-745411547